src/bundle.c

```diff
@@ -420,8 +420,10 @@ static int install_bundles(struct list *bundles, struct list **subs, int current
 	struct list *iter;
 	struct list *to_install_bundles = NULL;
 	struct list *to_install_files = NULL;
+	struct time times[5] = {0};
 
 	/* step 1: check bundle args are valid if so populate subs struct */
+	grab_time(START, times, 0, "Add bundles and recurse");
```

Comment:
For ease of use, it would be nice to omit the third argument if possible... So, `grab_time(START, times, "description")`.